pom.xml

```diff
@@ -5,14 +5,14 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>ru.itis</groupId>
-    <artifactId>wish_list</artifactId>
+    <artifactId>wish-list</artifactId>
     <version>0.1</version>
     <packaging>jar</packaging>
 
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.0.5.RELEASE</version>
+        <version>2.2.5.RELEASE</version>
```

Comment:
What's the reason?

Reply:
https://github.com/spring-projects/spring-boot/issues/20217